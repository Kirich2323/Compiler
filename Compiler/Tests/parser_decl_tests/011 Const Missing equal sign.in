const
    a = 10;
    b 0.5;