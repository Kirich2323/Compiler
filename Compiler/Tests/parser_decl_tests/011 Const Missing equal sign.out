(3 ; 7): Got unexpected token: "real number". Expected: "=".